vexflow/vexflow.d.ts

```diff
@@ -6,7 +6,7 @@
 //inconsistent namespace: this is a helper funtion from tables.js and should not pollute the global namespace!
 declare function sanitizeDuration(duration : string) : string;
 
-declare namespace Vex {
+declare module Vex {
```

Comment:
You should use `namespace` instead of internal `module`.
https://github.com/Microsoft/TypeScript/issues/2159